docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+### Into your app
+
+Linking to your app from other apps or a web browser is achieved using a URL based on the scheme for your app. These URLs are called **deep links**.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
+### Android App Links
+
+Android App Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to Android devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website.
+
+<BoxLink
+  title="Configure Android App Links"
+  description={
+    <>
+      Learn how to configure <CODE>intentFilters</CODE> and set up two-way association from a
+      standard web URL.
+    </>
+  }
+  href="/linking/android-app-links/"
+  Icon={AndroidIcon}
+/>
+
+### iOS Universal Links
+
+iOS Universal Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to iOS devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website. You can further configure the website by displaying a banner to the user to open your app using [Apple Smart Banner](/linking/ios-universal-links/#apple-smart-banner).
+
+<BoxLink
+  title="Configure iOS Universal Links"
+  description={
+    <>
+      Learn how to configure <CODE>associatedDomains</CODE> and set up two-way association.
+    </>
+  }
+  href="/linking/ios-universal-links/"
+  Icon={AppleIcon}
+/>
+
+## Use Expo Router to handle deep linking
+
+To implement any of the above Linking strategy, **we recommend using** [Expo Router](/router/introduction/) since deep linking is automatically enabled for all of your app's screens.
+
+**Benefits:**
+
+- When implementing deep links, you can skip [adding `scheme` to your app's config](/linking/into-your-app/#add-a-custom-scheme-in-app-config).
+- `Link` component from Expo Router can be used to [handle URL schemes to other apps](/linking/into-other-apps/#expo-router)
+- Android App Links and iOS Universal Links require configuring two-way association and runtime routing for the link in your app. This configuration is done in JavaScript and must be applied to every route to ensure coordination between web and native platforms. Since Expo Router enables deep links automatically, you don't have to configure runtime routing separately.
```

Comment:
Also fuzzy on this being a router-specific benefit ... Router requires the same universal link configuration in order for universal links to work. I guess one indirect benefit is that if your bespoke non-router link handling matched on the scheme as well as the deep link, that would break with a universal link (or any other scheme, for that matter).

Reply:
This piece of content was already there in both sections (App and Universal links) as a note and was presented as a benefit of Expo Router. Instead of repeating ourselves in both documents, I decided to add this here as one place to gather all the info.

Based on your feedback, should I re-word this to say "hey, this is an indirect benefit of using Expo Route". On my research, I was also comparing this React navigation where you have to define a prefix: https://reactnavigation.org/docs/deep-linking/. iiuc, this is handled by Expo Router automatically.